apis/core/bootstrap/shapes/dimension.ts

```diff
@@ -83,18 +123,92 @@ ${shape('dimension/metadata')} {
       ${sh.minLength} 1 ;
       ${dash.singleLine} false ;
       ${sh.order} 30 ;
-    ], [
+    ] , [
+      ${sh.name} "Data kind" ;
+      ${sh.path} ${cube.dataKind} ;
+      ${sh.maxCount} 1 ;
+      ${sh.nodeKind} ${sh.BlankNode} ;
+      ${sh.node} [
+        ${sh.property} [
+          ${sh.name} "Choose type" ;
+          ${sh.path} ${rdf.type} ;
+          ${sh.minCount} 1 ;
+          ${sh.maxCount} 1 ;
+          ${sh.nodeKind} ${sh.IRI} ;
+          ${dash.editor} ${dash.EnumSelectEditor} ;
+          ${sh.in} (
+            ${schema.GeoCoordinates}
+            ${schema.GeoShape}
+            ${time.GeneralDateTimeDescription}
+          ) ;
+          ${sh.order} 10 ;
+        ] , [
+          ${sh.name} "Time precision" ;
+          ${sh.path} ${time.unitType} ;
+          ${sh.maxCount} 1 ;
+          ${sh.nodeKind} ${sh.IRI} ;
+          ${sh1.if} [
+            ${sh.path} ${rdf.type} ;
+            ${sh.hasValue} ${time.GeneralDateTimeDescription} ;
+          ] ;
+          ${sh.in} (
+            ${time.unitYear}
+            ${time.unitMonth}
+            ${time.unitWeek}
+            ${time.unitDay}
+            ${time.unitHour}
+            ${time.unitMinute}
+            ${time.unitSecond}
+          ) ;
+          ${sh.order} 20 ;
+        ] ;
+      ] ;
+      ${sh.maxCount} 1 ;
+      ${sh.order} 40 ;
+    ] , [
       ${sh.path} ${schema.about} ;
       ${sh.nodeKind} ${sh.IRI} ;
       ${sh.minCount} 1 ;
       ${sh.maxCount} 1 ;
       ${dash.hidden} true ;
-    ]
+    ] ,
+    ${validateDataKindShape}
   .
 
   ${qudt.NominalScale} ${rdfs.label} "Nominal"@en .
   ${qudt.OrdinalScale} ${rdfs.label} "Ordinal"@en .
   ${qudt.IntervalScale} ${rdfs.label} "Interval"@en .
   ${qudt.RatioScale} ${rdfs.label} "Ratio"@en .
+
+  ${schema.GeoCoordinates}
+    ${rdfs.label} "Geographic coordinates"@en ;
+  .
+  ${schema.GeoShape}
+    ${rdfs.label} "Geographic shape"@en ;
+  .
+  ${time.GeneralDateTimeDescription}
+    ${rdfs.label} "Time description"@en ;
```

Comment:
Label suggestions welcome

Reply:
I will ask the customer on the labels. Good for now.